requirements/edx/github.in

```diff
@@ -67,7 +67,6 @@
 #   * Alphabetize dependencies by DIST-NAME.
 git+https://github.com/openedx/blockstore.git@1.2.5#egg=blockstore==1.2.5
 git+https://github.com/openedx/codejail.git@3.3.0#egg=edx-codejail==3.3.0
-git+https://github.com/openedx/codejail.git@3.1.3#egg=codejail==3.1.3
```

Comment:
Reviewers: Codejail was accientally specified twice. Once as `codejail==3.1.3`, and again as `edx-codejail==3.3.0`. The **latter** is the correct one.

This PR removes `codejail==3.1.3` from all requirements files. If you check, you can see that `edx-codejail==3.3.0` will still be installed.